### ISSUE TITLE
[BUG] pi sugar 3 plus webUI does not work

### ISSUE THREAD

**Issue by Tetsujinfr** [PiSugar/pisugar-power-manager-rs#58]
**Environment**
 - Pi hardware version: <3b+...>
 - OS version: <Bullseye>
 - Application version: <where do I find this???>

**To Reproduce**
1. Install pi sugar 2 pro
2. Install software as per doc on wiki and get the server running, all good.
3. Remove pi sugar 2 pro and replace by pi sugar 3 plus
4. Go directly to webUI (I did not find any specific softwate install instructions for this device)
5. Still see pi sugar pro 2 web UI, with no battery status update 

**Expected behavior**
Pi sugar 3 plus webUI should work right of the bat no? Do I need special software for this hardware as opposed to pisugar pro 2?

Please guide me to specific software install instructions for pi sugar 3 plus. And if I need to uninstall pi sugar 2 pro software fist, let me know and tell me how please.
Thanks

**Comment by Tetsujinfr** [comment]
Fyi I can see the 2 i2c addresses of the pis3+ device, but the WEBUI is desperately not active.
I still have the pi sugar 2 pro software installed on my raspberry3 b+.
Here is a screenshot:
![screenshot_pisugar3plus_no_WEBUI](https://user-images.githubusercontent.com/29961693/158803046-e78ef7ad-835b-4c61-b791-d9b13a03b245.png)
:

**Comment by Tetsujinfr** [comment]
Ok so I managed to solve this issue.
I did follow the [instructions](https://github.com/PiSugar/pisugar-power-manager-rs) of this project repo, i.e. unistall the existing packages and re-install the power management rs stack, and it works now.
There is some logic to it but I think that:
1. the instructions on the flyer card in the pisugar box mentioning to go to github/pisugar/pisugar/wiki are actually misleading, it should rather point to this repository (or the install instructioçns on the pi sugar3 wiki should made clearer)
2. it took me some time to make sure I understand that WebUI and pisugar-power-manager-rs are the same thing

Great product / software, but documentation could be significantly made clearer imo

closing this issue
